### DISCUSSION TITLE
Using the Google Translate in Automatic Translation

### DISCUSSION THREAD

**Discussion by vitorpadilha** [exoplatform/exo-community#52] [Q&A]
Hi,

I generate the API Key in Google Console and put it in eXo Platform but the itens has not been translate. This functionality was disponible to Whole Site??

Thank you.

**Comment by rdenarie** [comment]
Hello @vitorpadilha 
Currently, only activities and comments can be translated.
To translate an activity, click on 3-dot menu of the activity, and choose "Translate" :
![image](https://github.com/exoplatform/exo-community/assets/807839/6e518873-cb1f-4f2b-99b3-bf49a9d51fa8)

In next version (6.5), automatic translation will be available for news and notes.

**Comment by jonasbrant** [comment]
The translation still just translating a few words, several of them still in english causing chalenges to local users from other native languages. 
One exemple is KUDOS that in portugues has a terrible association and we have been dificult to find ways to translate. 
![image](https://github.com/exoplatform/exo-community/assets/11363238/0542842e-a53c-4d6c-85fc-20250851605a)

**Reply by rdenarie** [reply]
Hi @jonasbrant 
Contrary to the initial poster, you speak about interface internationalisation.

For this, we use crowdin tool, which allow contributors to provides translation for each elements of the interface.

If you want you contribute here : https://crowdin.com/project/exo-platform/pt-BR#/Add-ons/kudos
(I select specifically kudos, and pt-BR language, but you can change).

As the initial poster question is answered, I close this discussions. If you need more informations, you cannot create an account on community.exoplatform.com an ask your question. You will have more audience and more chances to get an answer
